NEWS.md

```diff
@@ -1,5 +1,7 @@
 # httr2 (development version)
 
+* `multi_dots()` now re-calculates n lengths when using `.multi = "explode"` to avoid select/recycling issues (@Kevanness, #719).
```

Comment:
```suggestion
* `req_url_query()` now re-calculates n lengths when using `.multi = "explode"` to avoid select/recycling issues (@Kevanness, #719).
```